recipes/emboss/meta.yaml

```diff
@@ -3,21 +3,21 @@ package:
   version: "6.5.7"
 
 build:
-  number: 2
+  number: 3
 
 source:
   fn: EMBOSS-6.5.7.tar.gz
   url: ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/EMBOSS-6.5.7.tar.gz
   md5: 6a2cb3f93d5e9415c74ab0f6b1ede5f0
 
 requirements:
-  build:
-   - gcc # [linux]
-   - llvm # [osx]
-   - python
-   - zlib
-  run:
-   - zlib
+    build:
```

Comment:
4-space indentation?

Reply:
yes, do we have a rule for this. I tried to do this in files with mixed indents ... 2 and 4 are allowed in yaml afaik.

Reply:
I was just surprised you changed it to 4 since the prevalent style is 2 space, I think. (So this and the other "indentation" remarks are just meant as comments, not necessarily change requests. Imprecise again..)